test/extensions/secondorder/hbp/kfac_settings.py

```diff
@@ -25,5 +35,23 @@
         "loss_function_fn": lambda: MSELoss(reduction="mean"),
         "target_fn": lambda: regression_targets((1, 1)),
         "id_prefix": "one-additional",
-    }
+    },
+    {
+        "input_fn": lambda: rand(3, 10),
+        "module_fn": lambda: Sequential(
+            Linear(10, 5),
+            ReLU(),
+            # skip connection
+            Parallel(
+                Linear(5, 5),
```

Comment:
Replace with `Identity` to make sure popular skip connections work

Reply:
Fixed this in commit
"Added identity extension to HBP"

Reply:
Closing